src/components/AddBook.js

```diff
@@ -1,16 +1,29 @@
 /* eslint-disable */
```

Comment:
- [ ] Linters are a great tool to make your code professional, clean, and readable, kindly remove all the disabling commands `/* eslint-disable */` from all of your pages including `BookItem.js`, `Books.js` , `Categories.js` etc ... and try to fix all the linters 🙏